New version of Kontent source plugin adjustments (#24047)
gatsbyjs/gatsby: docs/docs/sourcing-from-kentico-kontent.md

You can also take advantage of Kentico Kontent's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Kontent, the [source plugin](https://github.com/Kentico/kontent-gatsby-packages/tree/master/packages/gatsby-source-kontent) will ensure the proper nodes are created for your Gatsby site.

_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Kontent works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Kontent Gatsby starter site](https://github.com/Kentico/gatsby-starter-kontent), which includes worked examples for querying content. Or you could also check out the [Kontent starter based on Lumen template](https://github.com/Kentico/gatsby-starter-kontent-lumen), which is a minimal, lightweight and mobile-first starter for creating blogs using Gatsby and Kentico Kontent._

## Setup

### Kentico Kontent

> The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-uputm_medium=gatsbyjsorg&utm_source=guide). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.

Reviewer: ```suggestion
The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-up?utm_medium=gatsbyjsorg&utm_source=guide). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.
```

Looks like this is missing a "?" 🤔

Author: d950fcf

When you rerun `gatsby develop`, you'll be able to see each article as a page with content pulled from Kentico Kontent. To see a list of all pages, visit `http://localhost:8000/asdf` (or any other url that generates a 404).

> The body copy for this article comes from a rich text element in Kentico Kontent. Links and inline linked items (e.g., embedded videos) are not resolved by default for rich text elements. If you want to resolve them, you can query the required data in structured form for resolution and create your own React components you could use [Rich text element component](https://github.com/Kentico/kontent-gatsby-packages/tree/master/packages/gatsby-kontent-components#rich-text-element-component) that is a part of the [@kentico/gatsby-kontent-components](https://www.gatsbyjs.org/packages/@kentico/gatsby-kontent-components) package.

Reviewer: ```suggestion
The body copy for this article comes from a rich text element in Kentico Kontent. Links and inline linked items (e.g., embedded videos) are not resolved by default for rich text elements. If you want to resolve them, you can query the required data in structured form for resolution and create your own React components. You could use [Rich text element component](https://github.com/Kentico/kontent-gatsby-packages/tree/master/packages/gatsby-kontent-components#rich-text-element-component) that is a part of the [@kentico/gatsby-kontent-components](https://www.gatsbyjs.org/packages/@kentico/gatsby-kontent-components) package.
```

> Since the Kontent source plugin in defining the GrphQL schema for data from Kentico Kontent, you could use this schema and extend it according to your needs. There are [some examples](https://github.com/Kentico/kontent-gatsby-packages/tree/master/site#examples) what you could do in you application.

Reviewer: ```suggestion
Since the Kontent source plugin is defining the GraphQL schema for data from Kentico Kontent, you could use this schema and extend it according to your needs. There are [some examples](https://github.com/Kentico/kontent-gatsby-packages/tree/master/site#examples) of what you could do in your application.
```
